jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/StandardDescriptorProcessorTest.java

```diff
@@ -137,7 +138,18 @@ public void testDuplicateServletMappingsFromDescriptors(WorkDir workDir) throws
         wac.setBaseResourceAsPath(docroot);
         wac.setDescriptor(webXml.toURI().toURL().toString());
         wac.start();
-        assertEquals("other", wac.getServletHandler().getServletMapping("/").getServletName());
+        ServletMapping[] mappings = wac.getServletHandler().getServletMappings();
+        ServletMapping mapping = null;
+        for (ServletMapping m : mappings)
+        {
+            if (m.containsPathSpec("/"))
+            {
+                assertEquals(null, mapping);
+                mapping = m;
+            }
+        }
```

Comment:
So you iterate through all of the mappings, what if there are 2 mappings both with path spec `/`? (I know this shouldn't be possible, but you found we accidentally did just that before this PR, and we should ensure that we don't have that regression again)

Reply:
Line 147 is doing exactly this.